### DISCUSSION TITLE
How can I narrow down clickable area of sections?

### DISCUSSION THREAD

**Discussion by TetraTheta** [hbstack/.github#46] [Q&A] [answered]
Maybe this would fit as an issue, rather than a discussion, but I also think this would be personal taste, so here it is.

---

At the top of [blog](https://github.com/hbstack/blog/blob/main/layouts/partials/hb/modules/blog/home/taxonomies.html) front page, there is a area for taxonomies like this:

![IMG1705335757](https://github.com/hbstack/.github/assets/37061801/7a132fb6-e516-41a1-8de5-8e2152d56f5e)

But I often find myself clicking and moving to these taxonomies because clickable area is too wide for its content.

![IMG1705335672](https://github.com/hbstack/.github/assets/37061801/8d398533-cc26-4180-8c36-bd16227b4561)

I want to narrow down these clickable area to be like red box at the image above. How can I do it?

These area doesn't cause problem with mobile environment, but cause problem in PC environment (and maybe tablet environment too). I don't think PC user will have problem navigating to these section if clickable area is narrowed down...

**Comment by razonyang** [comment]
You can either 1) wrap the <a> tag inside other tag or 2) remove the nav-fill.

I haven't tested, not sure if it can help.

**Comment by TetraTheta** [comment, marked as answer]
```css
.nav {
  justify-content: space-evenly;
}
.nav-fill > .nav-link {
  flex: initial;
}
```
Using this was enough for me. It is not perfect, but I'm OK with it.
